daemon/mgr/spec_linux.go

```diff
@@ -71,6 +71,22 @@ func populatePlatform(ctx context.Context, c *Container, specWrapper *SpecWrappe
 		s.Linux.MaskedPaths = nil
 	}
 
+	// Apply masked paths if specified.
+	if c.MaskedPaths != nil {
+		s.Linux.MaskedPaths = nil
+		for _, path := range c.MaskedPaths {
+			s.Linux.MaskedPaths = append(s.Linux.MaskedPaths, path)
```

Comment:
Actually no judgement or initialization is needed.

```go
for _, path := range c.MaskedPaths {
      s.Linux.MaskedPaths = append(s.Linux.MaskedPaths, path)
}
```
That's enough.

Reply:
when we createDefaultSpec, there are some default values for `MaskedPaths`, so we should make that nil firstly.

Reply:
prefer slice copy here.
```go
	s.Linux.MaskedPaths = make([]string, len(c.MaskedPaths))
	copy(s.Linux.MaskedPaths, c.MaskedPaths)
```